examples/foreach/foreach.bal

```diff
@@ -76,6 +76,5 @@ public function main() {
     foreach var i in 1..<endValue {
         sum = sum + i;
     }
-    io:println("summation from 1 to " + endValue + " excluding "
-                    + endValue + " is " + sum);
+    io:println("summation from 1 to ", endValue, " excluding ", endValue, " is ", sum);
```

Comment:
We have a max column count of 80 for proper rendering of examples. https://github.com/ballerina-platform/ballerina-lang/blob/master/docs/bbe-guidelines/bbe-guidelines.md

IINM, this exceeds 80?